<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreWidget</class>
 <widget class="QWidget" name="ScoreWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>88</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="geLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>17</y>
     <width>20</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="baiLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>17</y>
     <width>20</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="shiLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>17</y>
     <width>20</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="propLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>5</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#ff80ff;
font:14px &quot;微软雅黑&quot;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
